### DISCUSSION TITLE
packetsender as server

### DISCUSSION THREAD

**Discussion by sarashan** [dannagle/PacketSender#366] [Q&A]
Hello Thank you for the support. I would like to configure packetsender using cli to start as a server and respond to a udp ping on a specific port. is that doable ?

**Comment by dannagle** [comment]
This has been requested several times.  See here #170 . 

It's been a long wait, but I am happy to report that this feature is planned to be released soon. Hopefully in the next release. Be sure to visit the sponsors mentioned in the release notes for helping to make this happen.

**Comment by dannagle** [comment]
This feature is now in the development branch
